_codux/boards/avatar/avatar.board.tsx

```diff
@@ -0,0 +1,23 @@
+
+export default createBoard({
+    name: 'Avatar',
+    Board: () => {
+        return (
+            <div className={styles.container}>
+                <Avatar imageSrc={undefined} altText="unknown user avatar" />
```

Comment:
It's not an unknown user avatar. The alt text is for screen readers. Do you want screen readers to read "unknown user avatar"? No you don't.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt